File: sdk/go/entity/recon_summary.go
package entity

import (
	"time"

	"github.com/shopspring/decimal"
)

// ReconSummary 对账汇总统计实体
type ReconSummary struct {
	SummaryDate  time.Time       `json:"summaryDate" gorm:"type:date"`
	TotalOrders  int             `json:"totalOrders"`
	SuccessCount int             `json:"successCount"`
	FailCount    int             `json:"failCount"`
	InitCount    int             `json:"initCount"`
	TotalAmount  decimal.Decimal `json:"totalAmount" gorm:"type:decimal(20,2)"`
	CreateTime   time.Time       `json:"createTime"`
	UpdateTime   time.Time       `json:"updateTime"`
}

File: sdk/node.js/entity/ReconSummary.js
/**
 * 对账汇总实体类
 */
class ReconSummary {
    constructor() {
        this.id = null;
        this.reconDate = null;         // 对账日期 (YYYY-MM-DD string)
        this.totalOrders = 0;          // 总订单数
        this.totalAmount = 0;          // 总金额
        this.successCount = 0;         // 成功笔数
        this.failCount = 0;            // 失败笔数
        this.exceptionCount = 0;       // 异常笔数
        this.createTime = null;
        this.updateTime = null;
    }
}

module.exports = ReconSummary;
